Makefile

```diff
@@ -147,7 +147,7 @@ run: crd
 
 .PHONY: run-debug
 run-debug: run
-run-debug: CLI_FLAGS = "--log-level=debug"
+run-debug: CLI_FLAGS = --log-level=debug --tracing-enabled=true
```

Comment:
When running `make run-debug`, the Operator will create traces for each reconciliation loop. Just need to have a Jaeger instance at localhost.